### ISSUE TITLE
Sign in, Sign out

### ISSUE THREAD

**Issue by tastrax** [lutraconsulting/input#453]
There appears to be no way to fully sign out of Input. For security reasons (loss of device/privacy of data) I would like to sign out of input after use. Even after signing out I can still get to home and add data to projects. It never really shuts down completely. Maybe this is only on testflight?

**Comment by saberraz** [comment]
If you press the icon of profile on top right in Projects pages (e.g. My Project), it should take you to the profile page and you can sign out.

You can also go to Settings and tap on your name and it will sign you out.

But I think this is not documented, so we will add it to documentation.

**Comment by tastrax** [comment]
Thanks, I found those, but even if I am signed out I can still add data to projects. This means anyone could pick up my device and add data. It also means I cant protect the privacy of the data.

**Comment by saberraz** [comment]
For uploading the changes, you need to be signed in. So, if you sign out, all the changes will remain local.

For accessing the data, I guess you should rely on the device encryption and good security/privacy practices. Most of applications for mobile (including emails, photos, documents) will be visible, once user logs in.

**Comment by tastrax** [comment]
I agree but the groups I work with have a range of users who access the same device. For many users we would want NO access to Input data at all, hence the ability to sign out to maintain absolute confidentiality of data in the app whilst they use the device for other objectives.
